core/src/main/java/nu/yona/server/device/service/DeviceService.java

```diff
@@ -48,7 +44,12 @@
 @Service
 public class DeviceService
 {
-	private static final String ANDROID_MIN_APP_VERSION = "1.0.0";
+	// When updating the minimum version, be sure to update the version code and the version.
+	// Minimal technical version codes, used to verify the minimum
+	private static final int ANDROID_MIN_APP_VERSION_CODE = 5;
```

Comment:
Are you sure nobody is using a lower version?

Reply:
Yes. The current released version is 39, so we can be confident that nobody uses 5 or earlier. In case someone is, then it would be right to given an error.

However... even version code 39 does not send the open app event. That's only recently added to the Android app, as part of 1.1.